test/test_components.py

```diff
@@ -2370,13 +2369,6 @@ def test_no_color(self):
         assert config["encoding"]["x"]["field"] == "Horsepower"
         assert config["encoding"]["x"]["title"] == "Horse"
         assert config["encoding"]["y"]["field"] == "Miles_per_Gallon"
-        assert config["selection"] == {
```

Comment:
This part of the config does not always appear, I think depending on what version of altair you are using. I think its okay to leave out.